packages/analytics-ma-browser/src/index.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable @typescript-eslint/unbound-method */
+import client from './browser-ma-client';
+export { createInstance } from './browser-ma-client';
+export const {
+  add,
+  flush,
+  getDeviceId,
+  getSessionId,
+  getUserId,
+  groupIdentify,
+  identify,
+  init,
+  logEvent,
+  remove,
+  reset,
+  revenue,
+  setDeviceId,
+  setGroup,
+  setOptOut,
+  setSessionId,
+  setTransport,
+  setUserId,
+  track,
+} = client;
+export { Revenue, Identify, runQueuedFunctions } from '@amplitude/analytics-browser';
```

Comment:
Should we get revenue and identify from core?

Reply:
runQueuedFunctions should probably be in common

Reply:
Because this package extends from browser SDK, that is the main reason to export from browser SDK.

Reply:
I'd recommend that we import where it's originally defined instead of import chaining